pkg/pillar/cmd/usbmanager/usbcontroller.go

```diff
@@ -224,7 +224,23 @@ func (uc *usbmanagerController) cancel() {
 func ioBundle2PassthroughRule(adapter types.IoBundle) passthroughRule {
 	var pr passthroughRule
 
-	if adapter.UsbAddr == "" && adapter.PciLong != "" {
+	countAddr := 0
+
+	if adapter.UsbAddr != "" {
+		countAddr++
+	}
+
+	if adapter.UsbProduct != "" {
+		countAddr++
+	}
+
+	if countAddr > 1 {
+		log.Warnf("ambiguous passthrough rule %s (usbaddr: %s) (usbproduct: %s) (pcilong: %s)",
+			adapter.Phylabel, adapter.UsbAddr, adapter.UsbProduct, adapter.PciLong)
+		return nil
+	}
+
+	if adapter.PciLong != "" && adapter.UsbAddr != "" && adapter.UsbProduct != "" {
 		pci := pciPassthroughRule{pciAddress: adapter.PciLong}
```

Comment:
This is unreachable code given your earlier check to now allow usbaddr and usbproduct at the same time.